bin/postgresql-setup.in

```diff
@@ -121,6 +121,14 @@ Environment:
   PGSETUP_DEBUG              Set to '1' if you want to see very verbose shell
                              debugging output."
 
+# Warning about possible glibc collation changes BZ#1668301
+GLIBC_COLLATION_WARN_STRING="\
+WARNING: If you've just upgraded your database from a previous major
+version of Fedora or RHEL, please run reindexdb against your databases.
+Core library collation data may have changed and this will invalidate
+database indexes.  Most recently in Fedora 28 and RHEL 8 there have been
```

Comment:
I'm thinking about this once more, what about `For example, in Fedora 28 and RHEL 8 ... `

I'd like to avoid the the **Most recently** wording;  it is a temporary statement, which would have to be updated soon with the next Fedora/RHEL major releases.

Reply:
I was thinking the same thing. Good point, thanks.